drivers/si70xx/si70xx.c

```diff
@@ -18,22 +18,29 @@
  * @}
  */
 
-
+#include <string.h>
 #include "xtimer.h"
 
+#include "si70xx_internals.h"
 #include "si70xx.h"
 
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#define SI70XX_I2C     (dev->params.i2c_dev)
+#define SI70XX_ADDR    (dev->params.address)
+
 /**
  * @brief   Utility method to perform and reconstruct a measurement.
  */
 static uint32_t si70xx_measure(const si70xx_t *dev, uint8_t command)
 {
     uint8_t result[2];
 
-    i2c_acquire(dev->i2c_dev);
-    i2c_write_byte(dev->i2c_dev, dev->address, command);
-    i2c_read_bytes(dev->i2c_dev, dev->address, result, 2);
-    i2c_release(dev->i2c_dev);
+    i2c_acquire(SI70XX_I2C);
+    i2c_write_byte(SI70XX_I2C, SI70XX_ADDR, command);
```

Comment:
Checks added. But I don't think it's worth returning directly since the return value should be the value read. I just display a debug message indicating an error.